addOns/scripts/src/main/java/org/zaproxy/zap/extension/scripts/ScriptConsoleOptionsPanel.java

```diff
@@ -139,6 +179,23 @@ private JPanel getCodeStylePanel() {
         return codeStylePanel;
     }
 
+    private JComboBox<String> getFontNameComboBox() {
```

Comment:
Does it worth adding a preview like in core?

Reply:
The font change is applied immediately, whereas the core one needs a restart. And the font names are rendered in the font itself. So, I think we can skip the preview box in this case.

Reply:
Right, but you need to close the Options dialogue first.